content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -248,17 +248,18 @@ However, the particular path specified in the custom recycler Pod template in th
 `volumes` part is replaced with the particular path of the volume that is being recycled.
 
 ### PersistentVolume deletion protection finalizer
-{{< feature-state for_k8s_version="v1.23" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.31" state="beta" >}}
 
 Finalizers can be added on a PersistentVolume to ensure that PersistentVolumes
 having `Delete` reclaim policy are deleted only after the backing storage are deleted.
 
-The newly introduced finalizers `kubernetes.io/pv-controller` and
-`external-provisioner.volume.kubernetes.io/finalizer`
-are only added to dynamically provisioned volumes.
+The newly introduced finalizers `external-provisioner.volume.kubernetes.io/finalizer`
+is added to both dynamically provisioned and statically provisioned CSI volumes.
 
-The finalizer `kubernetes.io/pv-controller` is added to in-tree plugin volumes.
-The following is an example
+The finalizer `kubernetes.io/pv-controller` is added to dynamically provisioned 
+in-tree plugin volumes and skipped for statically provisioned in-tree plugin
+volumes.
```

Comment:
In addition to the changes made, we may want to say a few words about the implications
to users. This page is a user-facing page rather than a design document.